js/main.js

```diff
@@ -19,8 +19,130 @@ function formSubmit(event) {
+
+  divColumn.appendChild(journalImg);
+
+  var divColumn2 = document.createElement('div');
+  divColumn2.setAttribute('class', 'column-half column-full');
+
+  divRow.appendChild(divColumn2);
+
+  var titleContent = document.createElement('h2');
+  titleContent.setAttribute('class', 'title');
+  titleContent.textContent = objform.title;
+
+  divColumn2.appendChild(titleContent);
+
+  var notesContent = document.createElement('p');
+  notesContent.setAttribute('class', 'notes');
+  notesContent.textContent = objform.notes;
+
+  divColumn2.appendChild(notesContent);
+
+  return liColumn;
+
+}
+
+var ul = document.querySelector('ul');
+var $placeholder = document.querySelector('.placeholder');
+
+function viewSwap(viewData) {
+  for (var i = 0; i < $allView.length; i++) {
+    var $dataView = $allView[i].getAttribute('data-view');
+    if (data.view === $dataView) {
+      $allView[i].className = 'view';
+    } else if (data.view !== $dataView) {
+      $allView[i].className = 'view hidden';
+    }
+  }
+}
+
+window.addEventListener('DOMContentLoaded', viewSwap);
+
+function contentLoaded(event) {
+  if (data.entries.length > 0) {
+    $placeholder.className = 'hidden';
+  }
+  for (var i = 0; i < data.entries.length; i++) {
+    var content = renderJournalEntry(data.entries[i]);
+    ul.appendChild(content);
+  }
+}
+
+window.addEventListener('DOMContentLoaded', contentLoaded);
+
+var $anchor = document.querySelector('.entries');
+var $allView = document.querySelectorAll('.view');
+
+function entriesTab(event) {
+  var $dataView = event.target.getAttribute('data-view');
+  for (var i = 0; i < $allView.length; i++) {
+    if ($allView[i].getAttribute('data-view') === $dataView) {
+      $allView[i].className = 'view';
+    } else if ($allView[i].getAttribute('data-view') !== $dataView) {
+      $allView[i].className = 'view hidden';
+    }
+  }
+  data.view = 'entries';
+}
+
+$anchor.addEventListener('click', entriesTab);
+
+var $buttonAnchor2 = document.querySelector('.button2');
+
+function newFormTabButton2(event) {
+  var $dataView = event.target.getAttribute('data-view');
+  for (var i = 0; i < $allView.length; i++) {
+    if ($allView[i].getAttribute('data-view') === $dataView) {
+      $allView[i].className = 'view';
+    } else if ($allView[i].getAttribute('data-view') !== $dataView) {
+      $allView[i].className = 'view hidden';
+    }
+  }
+  data.view = 'entry-form';
+}
+$buttonAnchor2.addEventListener('click', newFormTabButton2);
+
+var $buttonAnchor = document.querySelector('.button1');
+
+function newFormTabButton1(event) {
+  var $dataView = event.target.getAttribute('data-view');
+  for (var i = 0; i < $allView.length; i++) {
+    if ($allView[i].getAttribute('data-view') === $dataView) {
+      $allView[i].className = 'view';
+    } else if ($allView[i].getAttribute('data-view') !== $dataView) {
+      $allView[i].className = 'view hidden';
+    }
+  }
+  data.view = 'entries';
+
+}
```

Comment:
These are all _very_ similar to your `viewSwap` function. Instead of repeating this behavior, you should just make use of your `viewSwap` function.